cibyl/files.py

```diff
@@ -0,0 +1,65 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+import logging
+import os
+from os import PathLike
+from typing import Iterable
+
+LOG = logging.getLogger(__name__)
+
+
+def _is_file_available(filename):
+    """
+    Checks if a file is available by trying to open to it.
+
+    Args:
+        filename(str): A path pointing to the file to be checked.
+    Returns:
+        (bool): Whether the file can be opened or not.
+    """
+    try:
```

Comment:
Isn't it enough to do something like os.path.isfile() instead of reading the whole file?

Reply:
Yes, I think that would be an easier solution. I was not aware of that, thanks.